src/main_wrapper.cc

```diff
@@ -0,0 +1,45 @@
+#include "openscad.h"
+
+#ifndef OPENSCAD_NOGUI
+#include <QtResource>
```

Comment:
Should probably be `#include <QtCore/qtresource.h>` to fly on all platforms

Reply:
This KDE code uses the same type of include: [#include \<QGuiApplication\>](https://github.com/DaddelZeit/kwin-effects-forceblur/blob/75db9f9de7b1049acbed14bd2ec3ea40cd3eacc3/src/blur.cpp#L28)

It should be possible then.

Reply:
Testing `#include <QtCore/qresource.h>` now.